tests/fsharpqa/Source/ErrorMessages/NameResolution/E_FieldNotInRecord.fs

```diff
@@ -0,0 +1,13 @@
+// #ErrorMessages #NameResolution 
+//<Expects status="error" span="(11,31-11,34)" id="FS1129">The record type 'F' does not contain a label 'Wall'\.</Expects>
```

Comment:
@KevinRansom @enricosada @dsyme I want to test for the multi line error message here. How can I do that?